paddle/fluid/platform/gloo_context.cc

```diff
@@ -0,0 +1,33 @@
+//   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
`2019` -> `2020`?

Reply:
Yes, I will fix the issue in the pr: https://github.com/PaddlePaddle/Paddle/pull/26586